tests/test_visitors/test_ast/test_classes/test_parent_class_visitor.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    WrongParentClassListDef,
+)
+from wemake_python_styleguide.visitors.ast.classes import (
+    WrongParentClassListVisitor,
+)
+
+
+@pytest.mark.parametrize('code', [
+    'class TestClassName(FirstClassName, SecondClassName, object): ...',
+])
+def test_wrong_class_definition_multiple_parent(
+    assert_errors,
+    parse_ast_tree,
+    code,
+    default_options,
+):
+    """Testing wrong class definition."""
+    tree = parse_ast_tree(code)
+
+    visitor = WrongParentClassListVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [WrongParentClassListDef])
+
+
+@pytest.mark.parametrize('code', [
+    'class TestClassName(object): ...',
+])
+def test_correct_class_definition_single_parent(
+    assert_errors,
+    parse_ast_tree,
+    code,
+    default_options,
+):
+    """Testing correct class definition with single parent."""
+    tree = parse_ast_tree(code)
+
+    visitor = WrongParentClassListVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('code', [
+    'class TestClassName(FirstTestClass, SecondTestClass): ...',
```

Comment:
You can use this test case for both this and `'class TestClassName(object): ...',` examples. They work the same way.